src/notifications/api/serializers.py

```diff
@@ -1,19 +1,197 @@
 import logging
+from json import dumps
 
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.serializers.json import DjangoJSONEncoder
+
+import requests
 from rest_framework import serializers
+from rest_framework.response import Response
+
+from notifications.conf.base import CHANNEL
```

Comment:
Always import settings via `django.conf.settings`

```suggestion
from django.conf import settings
```

Make sure to change the `CHANNEL` references to `settings.CHANNEL`.